healthcare/api-client/v1/fhir/fhir_stores_test.py

```diff
@@ -0,0 +1,270 @@
+                )
+            else:
+                raise
+
+    create()
+
+    yield
+
+    # Clean up
+    @retry(
+        wait_exponential_multiplier=1000,
+        wait_exponential_max=10000,
+        stop_max_attempt_number=10,
+        retry_on_exception=retry_if_server_exception,
+    )
+    def clean_up():
+        try:
+            fhir_stores.delete_fhir_store(
+                project_id, cloud_region, dataset_id, fhir_store_id
+            )
+        except HttpError as err:
+            # The API returns 403 when the FHIR store doesn't exist.
+            if err.resp.status == 404 or err.resp.status == 403:
+                print(
+                    "Got exception {} while deleting FHIR store".format(err.resp.status)
+                )
+            else:
+                raise
+
+    clean_up()
+
+
+@pytest.fixture(scope="module")
+def crud_fhir_store_id():
+    yield fhir_store_id
+
+    # Clean up
+    @retry(
+        wait_exponential_multiplier=1000,
+        wait_exponential_max=10000,
+        stop_max_attempt_number=10,
+        retry_on_exception=retry_if_server_exception,
+    )
+    def clean_up():
+        try:
+            fhir_stores.delete_fhir_store(
+                project_id, cloud_region, dataset_id, fhir_store_id
+            )
+        except HttpError as err:
+            # The API returns 403 when the FHIR store doesn't exist.
+            if err.resp.status == 404 or err.resp.status == 403:
+                print(
+                    "Got exception {} while deleting FHIR store".format(err.resp.status)
+                )
+            else:
+                raise
+
+    clean_up()
+
+
+def test_crud_fhir_store(test_dataset, capsys):
+    fhir_stores.create_fhir_store(project_id, cloud_region, dataset_id, fhir_store_id)
+
+    fhir_stores.get_fhir_store(project_id, cloud_region, dataset_id, fhir_store_id)
+
+    fhir_stores.list_fhir_stores(project_id, cloud_region, dataset_id)
+
+    fhir_stores.delete_fhir_store(project_id, cloud_region, dataset_id, fhir_store_id)
+
+    out, _ = capsys.readouterr()
+
+    # Check that create/get/list/delete worked
+    assert "Created FHIR store" in out
+    assert "name" in out
+    assert "fhirStores" in out
+    assert "Deleted FHIR store" in out
+
+
+def test_get_fhir_store_metadata(test_dataset, test_fhir_store, capsys):
+    fhir_stores.get_fhir_store_metadata(
+        project_id, cloud_region, dataset_id, fhir_store_id
+    )
+
+    out, _ = capsys.readouterr()
+
+    assert "version" in out
+
+
+def test_patch_fhir_store(test_dataset, test_fhir_store, capsys):
+    fhir_stores.patch_fhir_store(project_id, cloud_region, dataset_id, fhir_store_id)
+
+    out, _ = capsys.readouterr()
+
+    assert "Patched FHIR store" in out
+
+
+def test_import_fhir_store_gcs(test_dataset, test_fhir_store, capsys):
+    storage_client = storage.Client()
+    bucket = storage_client.get_bucket(gcs_uri)
+    blob = bucket.blob(source_file_name)
```

Comment:
I think it is very easy to create a fixture for blob. WDYT?
With the current code, it is not guaranteed to be deleted.

Reply:
Created a fixture.